utils/randomness.py

```diff
@@ -2,81 +2,5 @@
 import fauxfactory
 
 
-def generate_random_local_ip():
-    return "10.{}.{}.{}".format(
-        fauxfactory.gen_integer(0, 255),
-        fauxfactory.gen_integer(0, 255),
-        fauxfactory.gen_integer(0, 255))
-
-
 def generate_lowercase_random_string(size=8):
```

Comment:
Sooooo...what do you think about removing this one? I don't see that the strings need to be lowercase where it is being used, but even if that is the case, I think I'd prefer the .lower() call being done where it's needed instead of (thanks to this PR) having an entire library for it. :)